modules/codegen-plugin/src/sbt-test/codegen-plugin/smithy-rules/build.sbt

```diff
@@ -0,0 +1,10 @@
+lazy val root = (project in file("."))
```

Comment:
new scripted test, verifying that generating code from `smithy-rules-engine` does not lead to compilation error.